src/components/results/common/global-filter/results-global-filter.tsx

```diff
@@ -255,116 +268,142 @@ function ResultsGlobalFilter({
-                                resultsGlobalFilterStyles.chipBox,
-                                !recent ? resultsGlobalFilterStyles.filterTypeBox : undefined
-                            )}
-                        >
-                            <Box sx={resultsGlobalFilterStyles.groupLabel}>
-                                <FormattedMessage id={'results.globalFilter.' + group} />
-                            </Box>
-                            {children}
-                            {!recent && numOfGroupOptions - DEFAULT_NB_OPTIONS_DISPLAYED > 0 && (
-                                <Chip
-                                    component="li"
-                                    label={'+ ' + (numOfGroupOptions - DEFAULT_NB_OPTIONS_DISPLAYED)}
-                                    size="small"
-                                    sx={getResultsGlobalFiltersChipStyle(group)}
-                                    onClick={() => setNumberOfOptions(new Map([...numberOfOptions, [group, -1]]))}
-                                />
-                            )}
-                        </Box>
-                    );
+                    return <Box key={'keyBoxGroup_' + group}>{children}</Box>;
                 }}
-                // renderOption : the chips that are in the boxes that is visible when we focus on the AutoComplete
-                renderOption={(props, option: GlobalFilter) => {
-                    const { children, color, ...otherProps } = props;
+                // renderOption : the checkboxes visible when we focus on the AutoComplete
+                renderOption={(props, option: GlobalFilter, { selected }) => {
+                    const { key, children, color, ...otherProps } = props;
+                    // recent selected options are not displayed in the recent tab :
+                    const hideOption = selected && option.recent;
                     return (
-                        <Chip
-                            {...otherProps}
-                            component="li"
-                            label={getOptionLabel(option)}
-                            size="small"
-                            sx={getResultsGlobalFiltersChipStyle(option.filterType)}
-                        />
+                        !hideOption && (
+                            <Box component="li" key={key} {...otherProps}>
+                                <ListItemButton selected={selected}>
+                                    <Checkbox
+                                        size="small"
+                                        icon={<CheckBoxOutlineBlankIcon />}
+                                        checkedIcon={<CheckBoxIcon />}
+                                        checked={selected}
+                                        sx={{ paddingLeft: '0px' }}
+                                    />
+                                    <OverflowTooltip label={getOptionLabel(option, translate) ?? ''} />
```

Comment:
```suggestion
                                    <OverflowableText label={getOptionLabel(option, translate) ?? ''} />
```
I have tested this component which is in commons-ui, it works although is not nickel

Reply:
I tested as well and decided to use it. It is indeed always better to use commons-ui.

https://github.com/gridsuite/gridstudy-app/pull/2739/commits/70b607478c16efb2cb239f01d898795038df0204